ansible/deploy.yml

```diff
@@ -76,6 +76,21 @@
         when: package is not defined
         tags: [package]
 
+      - name: Download remote package
+        get_url:
+          url: "{{ package }}"
+          dest: "{{ packages_path }}/{{ package|basename }}"
+          checksum: "{{ package_checksum|default(omit) }}"
```

Comment:
It looks like package_checksum not used though I understand this enables checking checksum if specified manually. Ok.

Reply:
Yes, not used right now, we don’t publish checksums yet